src/components/icon/icons/Amex.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
 const AmexIcon = () => (
 	<g fill="none" fill-rule="evenodd">
```

Comment:
Those ESLint errors were bringing up things that do need to be fixed. `fill-rule` is not a proper React HTML attribute and may not be rendered correctly. React expects camel cased attribute names, so it should be `fillRule`.